packages/generators/app/src/resources/files/js/config/features.js

```diff
@@ -0,0 +1,3 @@
+module.exports = ({ env }) => ({
```

Comment:
I would not generate this file by default for new projects, we don't want people using it unless they explicitly go looking for it. It would just be confusing for most users.

Reply:
Yes I agree, I think i forgot to write this.